wp-info.json

```diff
@@ -0,0 +1,7 @@
+{
+  "db-name": "infinum_boilerplate_dev",
+  "db-user": "wp",
+  "db-pass": "wp",
+  "wp-user": "ivan.grginov",
```

Comment:
I'm fine with having your name, but it would be safer to use something totally generic, like `infinum.user`